dataland-frontend/src/components/forms/parts/elements/derived/ActivityTree.ts

```diff
@@ -0,0 +1,970 @@
+import { type TreeNode } from "primevue/tree";
+
+export const activityTree: Array<TreeNode> = [
+  {
+    name: "Forestry",
+    key: "1",
+    sector: "Forestry",
+    type: "header",
+    children: [
+      {
+        name: "Afforestation",
+        value: "Afforestation",
+        key: "1.1",
+        sector: "Forestry",
+        type: "child",
+        nace_codes: "A2",
+      },
+      {
+        name: "Rehabilitation and restoration of forests, including reforestation and natural forest regeneration after an extreme event",
+        value:
+          "RehabilitationAndRestorationOfForestsIncludingReforestationAndNaturalForestRegenerationAfterAnExtremeEvent",
+        key: "1.2",
+        sector: "Forestry",
+        type: "child",
+        nace_codes: "A2",
+      },
+      {
+        name: "Forest management",
+        value: "ForestManagement",
+        key: "1.3",
+        sector: "Forestry",
+        type: "child",
+        nace_codes: "A2",
+      },
+      {
+        name: "Conservation forestry",
+        value: "ConservationForestry",
+        key: "1.4",
+        sector: "Forestry",
+        type: "child",
+        nace_codes: "A2",
+      },
+    ],
+  },
+  {
+    name: "Environmental protection and restoration activities",
+    key: "2",
+    sector: "Environmental protection and restoration activities",
+    type: "header",
+    children: [
+      {
+        name: "Restoration of wetlands",
+        value: "RestorationOfWetlands",
+        key: "2.1",
+        sector: "Environmental protection and restoration activities",
+        type: "child",
+        nace_codes: null,
+      },
+    ],
+  },
+  {
+    name: "Manufacturing",
+    key: "3",
+    sector: "Manufacturing",
+    type: "header",
+    children: [
+      {
+        name: "Manufacture of renewable energy technologies",
+        value: "ManufactureOfRenewableEnergyTechnologies",
+        key: "3.1",
+        sector: "Manufacturing",
+        type: "child",
+        nace_codes: "C25, C27, C28",
```

Comment:
Why is this a string and not an array?

Reply:
We take this data from:
https://ec.europa.eu/sustainable-finance-taxonomy/assets/taxonomy.json
and there is "nace_codes": "C25, C27, C28" so we may the same.

Reply:
The field is only used once and as an array. So like in the comment above we could just reformat it in the dev tools and make the file how we actually need it instead of working around it.

Reply:
changed to array